.github/workflows/main.yml

```diff
@@ -362,12 +370,16 @@ jobs:
         SHELL: "/bin/bash" # To prevent https://github.com/kubernetes/minikube/issues/6643
       runs-on: ubuntu-18.04
       steps:
-      - name: install lz4
+      - name: Install lz4
         shell: bash
         run: |
           sudo apt-get update -qq
           sudo apt-get -qq -y install liblz4-tool
-      - name: install podman
+      - name: Install conntrack
```

Comment:
1- 
Install conntrack should only happen for none driver,
(the docker and podman should have it in their image)

2- lets add a comment also why we need conntrack for none (
mention the kube version ... needs it)